x/concentrated-liquidity/genesis.go

```diff
@@ -64,11 +81,34 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *genesis.GenesisState {
 		if err != nil {
 			panic(err)
 		}
+		accumObject, err := k.getFeeAccumulator(ctx, poolI.GetId())
```

Comment:
Not seeing where we export incentives accumulators.

Can you please add them as well to export / import genesis?

I think we can get them using this method:
https://github.com/osmosis-labs/osmosis/blob/3aa3dffae2f6e12ee931c8891d7f90cd49b97dc8/x/concentrated-liquidity/incentives.go#L54

Reply:
Gotcha, didnt know uptime accumulator == incentives accumulator, added along with necessary test cases